axonserver/src/main/java/io/axoniq/axonserver/localstorage/LocalEventStore.java

```diff
@@ -115,17 +114,18 @@ public LocalEventStore(EventStoreFactory eventStoreFactory, MeterRegistry meterF
     public LocalEventStore(EventStoreFactory eventStoreFactory,
                            MeterFactory meterFactory,
                            StorageTransactionManagerFactory storageTransactionManagerFactory,
-                           EventStoreExistChecker eventStoreExistChecker,
+                           Optional<EventDecorator> eventDecorator,
```

Comment:
I'd rather use a default implementation that does not decorates events instead of `Optional` as an input parameter (https://stackoverflow.com/questions/31922866/why-should-java-8s-optional-not-be-used-in-arguments).